### PR TITLE
add JoyTokenIcon component

### PR DESCRIPTION
**Pull request by kdembler**

### CODE REVIEW

File: packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  SvgJoyTokenPrimary32,
+  SvgJoyTokenPrimary48,
+  SvgJoyTokenSilver16,
+  SvgJoyTokenSilver24,
+  SvgJoyTokenSilver32,
+  SvgJoyTokenSilver48,
+} from '@/components/_icons'
+
+type JoyTokenIconVariant = 'primary' | 'silver' | 'regular'
+type JoyTokenIconSize = 16 | 24 | 32 | 48
```
Comment (eldiegod) [inline packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx:21]: wouldn't strings make more sense?

**Review by eldiegod — APPROVED** [review]
it's working well and small comment

File: packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx (hunk 1 above)
Comment (kdembler) [reply packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx:21]: Is there any advantage to using string?

Comment (eldiegod) [reply packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx:21]: At least for vscode better intellisense/autocomplete

Comment (kdembler) [reply packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx:21]: Seems to work fine:
Screenshot 2022-03-02 at 17 07 46

**Review by toiletgranny — APPROVED** [review]

**Pull request merged**